test/Tests/src/Semantic/Meta_Spec.enso

```diff
@@ -222,13 +223,16 @@ spec =
             Meta.is_atom typ . should_be_false
             meta_typ = Meta.meta typ
             meta_typ . should_be_a Meta.Type
+            meta_typ.short_name . should_equal "My_Type"
             cons = case meta_typ of
                 Meta.Type.Value _ -> meta_typ.constructors
                 _ -> Test.fail "Should be a Meta.Type.Value: " + meta_typ.to_text
 
             cons.length . should_equal 1
             cons.at 0 . should_be_a Meta.Constructor
             cons . map (x -> x.name) . sort . should_equal [ "Value" ]
+            cons.each ctor->
+                ctor.parent_type . should_equal meta_typ
```

Comment:
Why it should be called `parent_type`? _"Parent"_ is meant for other things: I can say: `Any` is parent type of `Text` and other types as well. Or `Number` is parent type of `Integer`. But constructor doesn't have a _"parent type"_.

Java is using [declaring class](https://docs.oracle.com/en/java/javase/17/docs/api/java.base/java/lang/reflect/Method.html#getDeclaringClass()).

Reply:
Ok, I can rename to `declaring_type`. Does that sound OK?